### PR TITLE
Green Bar

### PR DESCRIPTION
**Pull request by florianpilz**
Due to the 5.0.0 release of `zope.testbrowser`, the current version of `z3c.etestbrowser` breakes when picking it up automatically. Also the Python 2.6 tests are failing, but since the support was dropped by `zope.testbrowser` I will just drop it here, too. (Followed by adding Python 3 support).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,10 @@
-sudo: false
-cache:
-  directories:
-    - .tox
+# Configuration script to Travis-CI
 language: python
```
Comment (mgedmin) [inline .travis.yml]: Let's not drop `sudo: false`

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
-    - TOXENV=py26
-    - TOXENV=py27
+python:
+    - 2.7
 install:
-    - travis_retry pip install tox
+    - pip install tox
 script:
-    - tox
+    - tox -e py${TRAVIS_PYTHON_VERSION//[.]/}
```
Comment (mgedmin) [inline .travis.yml]: Or `pip install tox-travis` and it does this for you automagically: https://pypi.python.org/pypi/tox-travis

File: .travis.yml [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
 notifications:
-    email: false
+    email: false
```
Comment (mgedmin) [inline .travis.yml]: Please do not remove the trailing newline.

File: setup.py [hunk 4]
```diff
@@ -75,7 +75,7 @@ def get_test_requires():
       install_requires=[
           'setuptools',
           'lxml >= 2.2',
-          'zope.testbrowser >= 4.0',
+          'zope.testbrowser >= 4.0, < 5.0',
```
Comment (mgedmin) [inline setup.py]: Ugh.  :(

Comment (mgedmin) [inline setup.py]: Just how hard will it be to make it work with the latest zope.testbrowser?

Comment (florianpilz) [inline setup.py]: Wanted to start on a clean slate first. Working on Python 3 on a branch: https://github.com/zopefoundation/z3c.etestbrowser/tree/python3 (Already achieved compatibility with new API of `zope.testbrowser` 5.0.0 😀 )

Comment (mgedmin) [inline setup.py]: You're amazing!

**Pull request merged**